docs/wagtail-pages.md

```diff
@@ -0,0 +1,211 @@
+
+from v1.models.base import CFGOVPage
+
+
+class BrowsePage(CFGOVPage):
+    secondary_nav_exclude_sibling_pages = models.BooleanField(default=False)
+
+    # …
+
+    sidefoot_panels = CFGOVPage.sidefoot_panels + [
+        FieldPanel('secondary_nav_exclude_sibling_pages'),
+    ]
+```
+
+Because `secondary_nav_exclude_sibling_page` is a boolean field, 
+this creates a checkbox on the "Sidebar" tab when editing a page.
+Checking or unchecking that checkbox will set the value of `secondary_nav_exclude_sibling_pages` when the page is saved.
+
+In our `LandingPage` (used in the [StreamFields example above](#streamfields), 
+the `header` StreamField 
+[is added to the `content_panels` as a `StreamFieldPanel`](https://github.com/cfpb/cfgov-refresh/blob/master/cfgov/v1/models/landing_page.py#L31):
+
+```python
+from wagtail.wagtailadmin.edit_handlers import StreamFieldPanel
+from wagtail.wagtailcore.fields import StreamField
+
+from v1.atomic_elements import molecules
+from v1.models import CFGOVPage
+
+
+class LandingPage(CFGOVPage):
+    header = StreamField([
+        ('hero', molecules.Hero()),
+        ('text_introduction', molecules.TextIntroduction()),
+    ], blank=True)
+
+    # …
+
+    content_panels = CFGOVPage.content_panels + [
+        StreamFieldPanel('header'),
+        # …
+    ]
+```
+
+## Parent / child page relationships
+
+Wagtail provides two attributes to page models that enable 
+[restricting the three types of subpages or parent pages](https://docs.wagtail.io/en/v1.13.4/topics/pages.html#parent-page-subpage-type-rules) 
+a particular page model can have. 
+
+For example, in our [interactive regulations page models](https://github.com/cfpb/cfgov-refresh/blob/master/cfgov/regulations3k/models/pages.py#L138) 
+we have a `RegulationLandingPage` that can be created anywhere in the page tree. 
+`RegulationLandingPage`, however, can only have two types of pages created within it: 
+`RegulationPage` and `RegulationSearchPage`. 
+This parent/child relationship is expressed by setting `subpage_types` on `RegulationLandingPage`
+and `parent_page_types` on `RegulationPage` and `RegulationSearchPage` 
+to either the model classes themselves or to a model name in the form `app_label.ModelName`:
+
+```python
+from v1.models import CFGOVPage
+
+
+class RegulationLandingPage(CFGOVPage):
+    subpage_types = ['regulations3k.RegulationPage', 'RegulationsSearchPage']
+
+
+class RegulationsSearchPage(CFGOVPage):
+    parent_page_types = ['regulations3k.RegulationLandingPage']
+    subpage_types = []
+
+
+class RegulationPage(CFGOVPage):
+    parent_page_types = ['regulations3k.RegulationLandingPage']
+    subpage_types = []
+```
+
+!!! note 
+    Note: we prevent child pages from being added to `RegulationPage` and `RegulationSearchPage` 
+    by setting `subpage_types` to an empty list.
+
+## Template rendering
+
+Sometimes new Wagtail page types will need to make customizations to their base template 
```

Comment:
Does this ever _not_ happen?

```suggestion
New Wagtail page types will usually need to make customizations to their base template 
```

Reply:
With a subclass of `Page`, you're right. With `CFGOVPage`, I'm honestly not sure. I like your wording better though.

Reply:
Well, `CFGOVPage` doesn't have its own template definition, so anything subclassing it has to provide its own, right? I guess if you're subclassing a _subclass_ of CFGOVPage, you may not need to. But if you're doing that and not needing to change the template, do you _really_ need to do that?